python/cuml/cuml/dask/common/input_utils.py

```diff
@@ -27,7 +27,6 @@
 from dask_cudf import Series as dcSeries
 from dask.dataframe import Series as daskSeries
 from dask.dataframe import DataFrame as daskDataFrame
-from cudf import Series
```

Comment:
`cudf` is not a `run:` dependency of `cuml-cpu`, so it can't be imported unconditionally here. This change and the one below uses the `gpu_only_import()` wrapped version instead.

https://github.com/rapidsai/cuml/blob/80b98b12e93cde6213111507fb2f920ffe29f4e3/python/cuml/cuml/dask/common/input_utils.py#L40